terraso_backend/apps/graphql/schema/schema.graphql

```diff
@@ -2334,9 +2334,110 @@ input DepthDependentSoilDataUpdateMutationInput {
   soilOrganicMatterTesting: SoilIdDepthDependentSoilDataSoilOrganicMatterTestingChoices
   sodiumAbsorptionRatio: Decimal
   carbonates: SoilIdDepthDependentSoilDataCarbonatesChoices
+  siteId: ID!
+  clientMutationId: String
+}
+
+type SoilDataBulkUpdatePayload {
+  errors: GenericScalar
+  results: [SoilDataBulkUpdateResultEntry!]!
+  clientMutationId: String
+}
+
+type SoilDataBulkUpdateResultEntry {
+  siteId: ID!
+  result: SoilDataBulkUpdateResult!
+}
+
+union SoilDataBulkUpdateResult = SoilDataBulkUpdateSuccess | SoilDataBulkUpdateFailure
```

Comment:
If there's one of these per entry, maybe make that clear in the name. Imo without looking at the code around it, this name kinda makes it sound like there's one per bulk update